contracts/ICS20Transfer.sol

```diff
@@ -71,6 +72,9 @@ contract ICS20Transfer is
     /// @inheritdoc IICS20Transfer
     bytes32 public constant TOKEN_OPERATOR_ROLE = keccak256("TOKEN_OPERATOR_ROLE");
 
+    /// @inheritdoc IICS20Transfer
+    bytes32 public constant ERC20_CUSTOMIZER_ROLE = keccak256("ERC20_CUSTOMIZER_ROLE");
```

Comment:
This could probably just have been the same role as the token operator role (except the name doesn't fit super well). 

One thing I just realized is that we have been partly conflating permissions and roles. For the most part, our roles have a single permission, but if the same entity always has the same roles, then they should probably just be a single role. We could probably just have had a single operator role here that we could easily expand new operational permissions for in the future (rather than having to keep adding roles for each of them).

Reply:
I think having more roles is not a bad thing. It is often requested by zygis too. I do think that having grant and revoke functions for each role is not ideal. I have thought of a potential solution to that but I don't wanna implement it in favor of #443

Reply:
There are some downsides to having so many roles, especially managing them and making sure they are updated correctly (removing someone with a bunch of roles is going to be an annoying error-prone chore).

I would rather we think through what roles are needed and divide that up based on what would essentially be the same group of people. If not we don't have a role system, we have a granular permission system. Which can work, but it is much harder to manage and reason about.

Reply:
I think for this update it is fine though?
We can re-do our roles in another PR?

Reply:
Yeah, sounds good